### PR TITLE
test: configure JS tests

### PR DESCRIPTION
**Pull request by adekbadek**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Sets up & adds some JS tests. This is a refactor, but for testing the code in `streamlined.js` had to be executed as a function, which makes for a huge diff – but it's just indentation.

### How to test the changes in this Pull Request:

Check out if tests are passing 👇 

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,6 +30,15 @@ jobs:
           name: Run JS linter
           command: npm run lint
 
+  test-js:
+    docker:
+      - image: circleci/node:16
```
Comment (miguelpeixe) [inline .circleci/config.yml:35]: This image is deprecated. We should use [`cimg/node`](https://github.com/CircleCI-Public/cimg-node).

**Review by miguelpeixe — CHANGES_REQUESTED** [review]
Looks good! Only one small request:

File: .circleci/config.yml (hunk 1 above)
Comment (adekbadek) [reply .circleci/config.yml]: Done in 8d429bfb78824c55e50a0b361f7476e5c5bd5e81

**Review by miguelpeixe — APPROVED** [review]
Thank you for revising! 🎉

**Pull request merged**